.github/workflows/build-pr.yml

```diff
@@ -13,9 +13,9 @@ jobs:
         go: [ "^1.20" ]
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
       - name: Setup Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
```

Comment:
I'm wondering if we need to pin the actions to a specific git commit if it is already tagged with the version number. Also, I wonder how we can watch for later releases.

Reply:
I remember @puerco commented on another PR in the parsers project mentioning that we wanted commit ID here. I thought that’s a convention being followed.

Reply:
This is the comment
https://github.com/opensbom-generator/parsers/pull/58#discussion_r1209873095